tests/simple/array_test.py

```diff
@@ -30,15 +34,14 @@ def simple_array(verbose=False):
     print_func(a.is_complex(), a.is_real(), a.is_double(), a.is_single())
     print_func(a.is_real_floating(), a.is_floating(), a.is_integer(), a.is_bool())
 
-
-    a = af.Array(host.array('i', [4, 5, 6]))
+    a = af.Array(host.array("i", [4, 5, 6]))
     display_func(a)
     print_func(a.elements(), a.type(), a.dims(), a.numdims())
     print_func(a.is_empty(), a.is_scalar(), a.is_column(), a.is_row())
     print_func(a.is_complex(), a.is_real(), a.is_double(), a.is_single())
     print_func(a.is_real_floating(), a.is_floating(), a.is_integer(), a.is_bool())
 
-    a = af.Array(host.array('I', [7, 8, 9] * 3), (3,3))
+    a = af.Array(host.array("I", [7, 8, 9] * 3), (3, 3))
```

Comment:
Subjective: I dislike double quotes for a "char"-like string.

Reply:
If AF lib won't be extended with modules for web-dev, so it's okay to change all quotes to single and in another case, it'd be better to use double. But what's the point of quotes mixing? It decreases readability and adds chaos a bit into a code style.